docs-kits/kits/DCM-Kit/adoption-view/qna.md

```diff
@@ -0,0 +1,113 @@
+|12|How much does it cost to use DCM?|In order to evaluate the actual cost of participating in the DCM use case you have to compare cost vs gain.<br/><br/> Consider the following gains:<br/>- Increased capacity utilization rate<br/>- Reduced round trip times for information exchange<br/>- Emerging network effects <br/>- Common technical and business semantics<br/>- Increased resiliency in the supply network<br/><br/> Consider the following costs:<br/>- Onboarding with an operating company <br/>- Reoccurring fees<br/>- Adopting the DCM processes<br/>- Adopting a DCM compatible application<br/>- Adopting Catena-X dataspace compatible core infrastructure<br/><br/>The [Catena-X FAQ](https://catena-x.net/en/faq) has some insights into this as well.|
+
+<br/><br/><br/><br/>
+
+## Accessibility
+
+|#|Question|Answer|
+|--|--|--|
+|1|What do I need to exchange data within DCM?|You need a Catena-X certified DCM solution, interfacing with an Eclipse Dataspace Connector (EDC) and a registration with an operating company, such as Cofinity-X.|
+|2|Is EDC the same as EDI (Electronic Data Interchange)? Will I send the same messages via the same EDI portal?|EDC and EDI are completely different and not interoperable. You will have to exchange data via a Catena-X certified solution.|
+|3|Do I need to connect my internal systems to the Catena-X dataspace?|It is not required to connect your internal systems to the Catena-X dataspace. The EDC acts as a gateway for all your communication to and from the dataspace.|
+|4|How is my data protected when I utilize DCM via the Catena-X dataspace?|From a business perspective:<br/>Shared data is protected, because companies and business applications participating in the dataspace have to be certified. As part of the on-boarding process companies do sign framework agreements providing a basis for legal protection.<br/><br/>From a technical perspective: The EDC enforces this legal basis, via policy enforcement and contract negotiation, ultimately leading to data sovereignty.|
+|5|I participate in another Catena-X use case, will it simplify my access to the DCM use case and maybe even reduce my onboarding costs?|Yes, it will. Nearly all Catena-X use cases share a common infrastructure. Having an EDC in place and being already registered with an operating company,having assigned BPNL and BPNS, as well as a working wallet will reduce the on-boarding time and cost for DCM|
+|6|How often are Catena-X standards and in particular the DCM standard updated?|Catena-X standards can be updated on a quarterly basis, with breaking changes being allowed once a year. The DCM expert group is committed to introducing new features to the DCM standard without incurring breaking changes.|
```

Comment:
We should add to the answer, that we intend to assure a backward compatibility for the two newest major releases.

Reply:
Agreed.
Fixed via commit https://github.com/eclipse-tractusx/eclipse-tractusx.github.io/pull/984/commits/9c81d5998a4f2eb817f7e5cfcc5115cf825f7aaf